esmvaltool/preprocessor/_area.py

```diff
@@ -228,28 +223,26 @@ def average_region(cube, coord1, coord2, operator='mean', fx_files=None):
     grid_areas = tile_grid_areas(cube, fx_files)
 
     if not fx_files and cube.coord('latitude').points.ndim == 2:
-        logger.error('average_region ERROR: fx_file needed to calculate grid '
+        logger.error('fx_file needed to calculate grid '
                      'cell area for irregular grids.')
         raise iris.exceptions.CoordinateMultiDimError(cube.coord('latitude'))
 
-    if not grid_areas.any():
+    if grid_areas is None or not grid_areas.any():
```

Comment:
Will this break if `grid_areas == None`? Ie: you can't do `None.any()`.

Reply:
No, if the first expression evaluates to True, the second one will not be executed, see https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not

Reply:
forgot to answer this earlier today: yes, the `or` operator takes care of that as B-man is asying

Reply:
Huh, how about that. Every day is a school day. Thanks guys.